django/api/viewsets/upload.py

```diff
@@ -21,125 +21,139 @@
 from api.models.data_fleets import DataFleets
 from api.models.hydrogen_fleets import HydrogenFleets
 from api.serializers.datasets import DatasetsSerializer
-from api.services.ldv_rebates import import_from_xls as import_ldv
-from api.services.hydrogen_fueling import import_from_xls as \
-    import_hydrogen_fueling
-from api.services.charger_rebates import import_from_xls as \
-    import_charger_rebates
-from api.services.scrap_it import import_from_xls as \
-    import_scrap_it
-from api.services.arc_project_tracking import import_from_xls as \
-    import_arc_project_tracking
-from api.services.data_fleets import import_from_xls as \
-    import_data_fleets
-from api.services.hydrogen_fleets import import_from_xls as \
-    import_hydrogen_fleets
 from api.services.minio import minio_get_object, minio_remove_object
-from api.services.public_charging import import_from_xls as \
-    import_public_charging
-from api.services.speciality_use_vehicle_incentives import \
-    import_from_xls as import_suvi
 from api.services.datasheet_template_generator import generate_template
+from api.services.spreadsheet_uploader import import_from_xls
+from api.constants import *
```

Comment:
I think we may want to do `import api.constants as constants` to avoid possible naming/reference collisions; you can then refer to stuff inside api.constants.py using `constants`; e.g. `constants.ARCProjectTracking`